### ISSUE TITLE
Error using negate condition operator

### ISSUE THREAD

**Issue by aberenguel** [sepinf-inc/IPED#1174]
Using IPED 4 (13ab36492).

When I use a query with negate condition operator, the file name appears as ERROR in the table.

![image](https://user-images.githubusercontent.com/2973359/174653971-89aed825-d707-4808-8a3e-253a95996ecf.png)

The log shows that trace
```
java.lang.UnsupportedOperationException
	at org.apache.lucene.search.highlight.WeightedSpanTermExtractor$DelegatingLeafReader.getFieldInfos(WeightedSpanTermExtractor.java:460)
	at org.apache.lucene.search.FieldExistsQuery.rewrite(FieldExistsQuery.java:115)
	at org.apache.lucene.search.highlight.WeightedSpanTermExtractor.extract(WeightedSpanTermExtractor.java:250)
	at org.apache.lucene.search.highlight.WeightedSpanTermExtractor.extract(WeightedSpanTermExtractor.java:135)
	at org.apache.lucene.search.highlight.WeightedSpanTermExtractor.getWeightedSpanTerms(WeightedSpanTermExtractor.java:530)
	at org.apache.lucene.search.highlight.QueryScorer.initExtractor(QueryScorer.java:204)
	at org.apache.lucene.search.highlight.QueryScorer.init(QueryScorer.java:173)
	at org.apache.lucene.search.highlight.Highlighter.getBestTextFragments(Highlighter.java:173)
	at dpf.sp.gpinf.indexer.desktop.TextHighlighter.getHighlightedFrags(TextHighlighter.java:64)
	at dpf.sp.gpinf.indexer.desktop.ResultTableModel.getValueAt(ResultTableModel.java:328)
	at java.desktop/javax.swing.JTable.getValueAt(JTable.java:2706)
	at java.desktop/javax.swing.JTable.prepareRenderer(JTable.java:5724)
	at java.desktop/javax.swing.JTable.getToolTipText(JTable.java:3382)
	at java.desktop/javax.swing.ToolTipManager.mouseMoved(ToolTipManager.java:617)
	at java.desktop/java.awt.AWTEventMulticaster.mouseMoved(AWTEventMulticaster.java:338)
	at java.desktop/java.awt.Component.processMouseMotionEvent(Component.java:6680)
	at java.desktop/javax.swing.JComponent.processMouseMotionEvent(JComponent.java:3360)
	at java.desktop/java.awt.Component.processEvent(Component.java:6404)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5011)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4560)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by lfcnassif** [comment]
Thanks @aberenguel for reporting this. It happened to me recently, I'll investigate soon...

**Comment by lfcnassif** [comment]
Switching to this now.

**Comment by lfcnassif** [comment]
@aberenguel I think this is fixed, please test commit [b2e0fd4](https://github.com/sepinf-inc/IPED/commit/b2e0fd47d806c4a8e6d9260db364dc24c2f49961)

**Comment by aberenguel** [comment]
> @aberenguel I think this is fixed, please test commit [b2e0fd4](https://github.com/sepinf-inc/IPED/commit/b2e0fd47d806c4a8e6d9260db364dc24c2f49961)

Just tested this commit. It worked! Thanks!

**Comment by lfcnassif** [comment]
Thanks for testing!
